src/ApiModules/interactions.ts

```diff
@@ -60,8 +60,8 @@ export default class InteractionsApiModule {
     await this.storeInteraction("tutorialFinished");
   }
 
-  public async tutorialClosed() {
-    await this.storeInteraction("tutorialClosed");
+  public async tutorialClosed(step: number) {
+    await this.storeInteraction("tutorialClosed", undefined, { step });
```

Comment:
_Maybe_ we also add **stepText?** here as we do in `tutorialStepViewed` interaction?

Basically the motivation behind storing both step number & text is that whenever we change the content of the tutorial (add, modify or remove new steps) we lose the capability to track it in interactions (step 5 from 2 months ago won't be the same as step 5 today). That's why I'm trying to store some additional text ("stepText") - this is basically a short description / identifier of the step. In example now we have step number 5 with text `"tutorial2-5"`. When we introduce changes to the tutorial we should update it to i.e. `"tutorial3-5"`. Then we're able to distinguish in interactions that both are step 5 but not really the same step.